__tests__/components/navbar.test.jsx

```diff
@@ -0,0 +1,15 @@
+import Navbar from '../../components/navbar';
+import React from 'react';
+import { SessionProvider } from 'next-auth/react';
+import { render } from '@testing-library/react';
+
+describe('Navbar rendering correctly', () => {
+  it('renders correctly', () => {
+    const component = render(
+      <SessionProvider session={{ user: { name: 'test user' } }}>
+        <Navbar />
+      </SessionProvider>
+    );
+    expect(component).toMatchSnapshot();
```

Comment:
Could we possibly switch this to something like this?  
```suggestion
    const domTree = renderer.create(<Navbar />).toJSON();
     expect domTree.toMatchSnapshot();
    );

```

This way, we know that what we are testing is really the DOM tree the component renders.

If this is not possible because of the need to wrap our Navbar inside of a `<SessionProvider .../>` we can keep your existing code while we refactor our code to not explicitly need the `<Navbar />` to be wrapped in a `<SessionProvider .../>`

Reply:
Not sur if it's possble to avoid using <SessionProvider /> since we are wrapping all the app in it without refactoring the code